packages/manager/tools/testcafe/pages/common/managerParent.js

```diff
@@ -0,0 +1,35 @@
+import { Selector, t } from 'testcafe';
+import { getPageUrl } from '../../utils/helpers';
+
+export default class ManagerParentPage {
+  constructor({ currentPageNameInUrl } = {}) {
+    this.currentPageNameInUrl = currentPageNameInUrl;
+    this.navbarUser = Selector('[data-navi-id="user-menu"]');
+    this.logoutLink = Selector('[data-navi-id="logout"]');
+  }
+
+  async confirmCurrentPage() {
+    for (let i = 0; i < 15; i += 1) {
+      // eslint-disable-next-line no-await-in-loop
+      const currentPage = await getPageUrl();
+      if (currentPage.includes(this.currentPageNameInUrl)) {
+        break;
+      }
+    }
+  }
+
+  async toggleAccountMenu() {
+    await t.expect(this.navbarUser.visible).ok();
+    await t.click(this.navbarUser);
+  }
+
+  async disconnectFromManager() {
```

Comment:
to remove ? this must be in the Role file

Reply:
that's two different use cases the disconnection link must work in the manager

Reply:
that's two different use cases the disconnection link must work in the manager